build.gradle

```diff
@@ -19,40 +19,78 @@ ext {
     tempDir = "${rootProject.buildDir}/tmp"
     graalDir = "$tempDir/graal"
 
-    // This file is used as a dummy output in place of the graalvm repository directory.
-    // This prevents buildGraal from re-triggering cloneAndPatchRepository.
+    // These files are used as a dummies in place of the graalvm repository directory.
+    // This prevents buildGraal from re-triggering clone/patch operations and breaking incremental builds.
```

Comment:
This comment explains the purpose of these files. Can you also add a few lines explaining why Gradle cannot monitor the input directories as usual? And at the end, can you mention the catches of this ad-hoc solution? For instance, when should someone force the clone/patch operations?